registry/server/migrations/20201228201314_route_meta.ts

```diff
@@ -2,7 +2,7 @@ import { Knex } from 'knex';
 
 export async function up(knex: Knex): Promise<any> {
     return knex.schema.table('routes', (table) => {
-        table.text('meta');
+        table.json('meta');
```

Comment:
not sure that we can change existing migrations... if you expect json next in the code, it might not be applied to real DBs